native/c/makefile

```
@@ -405,19 +406,66 @@ $(OUT_DIR)/libztso.a: $(OUT_DIR)/ztso.o $(OUT_DIR)/libzut.a
 
 libztso.a: $(OUT_DIRS) $(OUT_DIR)/libztso.a
 
+#
+# Commands
+#
+$(OUT_DIR)/commands/console.o: commands/console.cpp
+	@echo 'Building $@'
+	@mkdir -p $(@D)
+	$(CXX) $(CPP_FLAGS) -c $< -o $@
+
+$(OUT_DIR)/commands/core.o: commands/core.cpp
+	@echo 'Building $@'
+	@mkdir -p $(@D)
+	$(CXX) $(CPP_FLAGS) -c $< -o $@
+
+$(OUT_DIR)/commands/ds.o: commands/ds.cpp
+	@echo 'Building $@'
+	@mkdir -p $(@D)
+	$(CXX) $(CPP_FLAGS) -c $< -o $@
+
+$(OUT_DIR)/commands/job.o: commands/job.cpp
+	@echo 'Building $@'
+	@mkdir -p $(@D)
+	$(CXX) $(CPP_FLAGS) -c $< -o $@
+
+$(OUT_DIR)/commands/tso.o: commands/tso.cpp
+	@echo 'Building $@'
+	@mkdir -p $(@D)
+	$(CXX) $(CPP_FLAGS) -c $< -o $@
+
+$(OUT_DIR)/commands/uss.o: commands/uss.cpp
+	@echo 'Building $@'
+	@mkdir -p $(@D)
+	$(CXX) $(CPP_FLAGS) -c $< -o $@
+
+$(OUT_DIR)/commands/tool.o: commands/tool.cpp
+	@echo 'Building $@'
+	@mkdir -p $(@D)
+	$(CXX) $(CPP_FLAGS) -c $< -o $@
```

These could be combined into a single block that builds all CPP files in the `commands` directory - I've been doing something similar in the middleware PR 😋 
```suggestion
$(OUT_DIR)/commands/%.o: commands/%.cpp
	@echo 'Building $@'
	@mkdir -p $(@D)
	$(CXX) $(CPP_FLAGS) -c $< -o $@
```